worlds/sc2wol/Items.py

```diff
@@ -153,13 +154,17 @@ def get_full_item_list():
 )
 
 
-item_name_groups = {"Missions":
-                        {"Beat Liberation Day", "Beat The Outlaws", "Beat Zero Hour", "Beat Evacuation",
+item_name_groups = {}
+for item, data in item_table.items():
+    if data.type not in item_name_groups:
```

Comment:
could be shortened by using dict.setdefault([]).append()

Reply:
Done